torch_frame/data/stats.py

```diff
@@ -37,19 +37,27 @@ def stats_for_stype(stype: torch_frame.stype) -> List['StatType']:
                 StatType.QUANTILES,
             ],
             torch_frame.categorical: [StatType.COUNT],
-            torch_frame.multicategorical: [StatType.MULTI_COUNT]
+            torch_frame.multicategorical: [StatType.MULTI_COUNT],
+            torch_frame.sequence_numerical: [
+                StatType.MEAN,
+                StatType.STD,
+                StatType.QUANTILES,
+            ]
         }
         return stats_type.get(stype, [])
 
     def compute(self, ser: Series, sep: Optional[str] = None) -> Any:
         if self == StatType.MEAN:
-            return np.mean(ser.values).item()
+            flattened = np.hstack(np.hstack(ser.values))
```

Comment:
Not sure if this is the best method, but it works with both `sequence_numerical` and `numerical`. 
If the `ser` represents `sequence_numerical`, 
then `ser.values` would look like `array([[list([1, 3.0])], [list([1, 2, 3.0])]], dtype=object)`, so we need to hstack twice.

Reply:
Looks good to me.